go-packages/meep-sandbox-ctrl-client/api/swagger.yaml

```diff
@@ -2326,38 +2316,24 @@ definitions:
       name:
         type: "string"
         description: "Application name"
-      type:
-        $ref: "#/definitions/ApplicationType"
-      state:
-        $ref: "#/definitions/ApplicationState"
       mepName:
```

Comment:
MEP is an ETSI MEC concept, can we name that parameter differently (more generically)
node-name?

Reply:
Fixed in PR #327